simpler_env/evaluation/evaluate.py

```diff
@@ -841,3 +876,90 @@ def run_comprehensive_evaluation(env_policy: AiroaBasePolicy, ckpt_path: str) ->
         "simulation_robust_score": sim_score,
         "visual_matching_robust_score": vm_score,
     }
+
+
+def run_partial_evaluation(env_policy: AiroaBasePolicy, ckpt_path: str, task: str) -> Dict[str, float]:
+    print("=" * 80)
```

Comment:
Just a suggestion: we can write something as follows to print multiple lines to output:

print("""
========================================
🚀 STARTING PARTIAL EVALUATION 🚀
Task: {task}
Checkpoint: {ckpt_path}
Weights: Sim={SIM_WEIGHT}, VisualMatching={VISUAL_MATCHING_WEIGHT}
========================================
""")

But this is not very urgent to the contest. For now feel free to ignore this suggestion.